easel/lib/screens/creator_hub/widgets/nfts_grid_view.dart

```diff
@@ -58,6 +62,47 @@ class NftGridViewItem extends StatelessWidget {
                 assetType: nft.assetType.toAssetTypeEnum(),
               ),
             )),
+        Container(
+          decoration: const BoxDecoration(
+            gradient: LinearGradient(colors: [Colors.black87,Colors.black54,Colors.black45,Colors.black38,
+              Colors.black26,Colors.black12,Colors.transparent],
+              begin: Alignment.topCenter,
+              end: Alignment.bottomCenter,
+            ),
+          ),
+          child: Row(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(1.h),
+                  color: context.read<CreatorHubViewModel>().selectedCollectionType == CollectionType.draft ?EaselAppTheme.kLightRed:
+                  nft.isEnabled && nft.amountMinted < int.parse(nft.quantity)?EaselAppTheme.kBlue:EaselAppTheme.kDarkGreen,
```

Comment:
Kindly format this screen. Looks like this line is not formatted.

Reply:
Sorry i did not get your point, can you please elaborate? format how?

Reply:
I think we need to format the whole file

Reply:
@Kamran-Khan-Gene you need to adjust max line, to 200 char in your editor